prov/gni/src/gnix_mr.c

```diff
@@ -44,6 +41,37 @@
 #include "gnix_mr.h"
 #include "gnix_priv.h"
 
+typedef enum cache_entry_flags {
+	GNIX_CE_RETIRED = 1 << 0,
```

Comment:
Not at all. I leave it there for consistency sake since any new flags
would be of the same form. The compiler will compute those values and
directly replace them, so the cost at runtime is nil. The only purpose
that it serves is readability so that we know which bit is affected.

New flags, if any will be needed, would be 1 << 1, 1 << 2, and so forth.

If this is an issue, we can discuss.
-- Jim




On 12/22/15, 9:47 AM, "e-harvey" <notifications@github.com> wrote:

>In 
>prov/gni/src/gnix_mr.c
><https://github.com/ofi-cray/libfabric-cray/pull/582#discussion_r48265451>
>:
>> @@ -44,6 +41,37 @@
>>  #include "gnix_mr.h"
>>  #include "gnix_priv.h"
>>  
>> +typedef enum cache_entry_flags {
>> +	GNIX_CE_RETIRED = 1 << 0,
>Is a left-shift by zero necessary?
>—
>Reply to this email directly or
>view it on GitHub 
><https://github.com/ofi-cray/libfabric-cray/pull/582/files#r48265451>.
>
>